test/extensions/filters/network/thrift_proxy/filters/ratelimit/ratelimit_test.cc

```diff
@@ -338,7 +338,7 @@ TEST_F(ThriftRateLimitFilterTest, ErrorResponseWithDynamicMetadata) {
   EXPECT_CALL(filter_callbacks_.stream_info_, setDynamicMetadata(_, _))
       .WillOnce(Invoke([&dynamic_metadata](const std::string& ns,
                                            const ProtobufWkt::Struct& returned_dynamic_metadata) {
-        EXPECT_EQ(ns, ThriftProxy::ThriftFilters::ThriftFilterNames::get().RATE_LIMIT);
```

Comment:
There's an #include of well_known_names.h at the top of this file that needs to be remove.

Reply:
> I think that step may have been missed in some of your other PRs as well.

Never mind that part. Just saw #16836.

Reply:
Thanks for review, I will fix them.